Services/UserService/Utils/EmailSender.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Net;
+using System.Net.Mail;
+using UserService.Interfaces;
+
+namespace UserService.Utils
+{
+    public static class EmailSender
+    {
+        public static void SendEmail(string email,ISecretTokenEngine secretTokenEngine)
+        {
+            var from = new MailAddress("traidplatform@mail.ru", "Trading Station");
+            var to = new MailAddress(email);
+            string secretToken =  secretTokenEngine.GetToken(email).ToString();
+            string link = $"https://localhost:5011/users/confirm?secretToken={secretToken}";
```

Comment:
I don't like this because user will get to a weird empty page, but ok

Reply:
I think we need to merge it and than I add page for GUI

Reply:
> I think we need to merge it and than I add page for GUI

Why can't you do it now?

Reply:
I don't find way to catch secrete token  in GUI from url